robocop/checkers/spacing.py

```diff
@@ -107,34 +158,85 @@
         ),
         rule_id="1012",
         name="consecutive-empty-lines",
-        msg="Too many empty lines ({{ empty_lines }}/{{ allowed_empty_lines }})",
+        msg="Too many consecutive empty lines ({{ empty_lines }}/{{ allowed_empty_lines }})",
         severity=RuleSeverity.WARNING,
+        docs="""
+        Example of rule violation::
+        
+            Keyword
+                Step 1
+                
+                
+                Step 2
+
+        """,
     ),
     "1013": Rule(
         rule_id="1013",
         name="empty-lines-in-statement",
         msg="Multi-line statement with empty lines",
         severity=RuleSeverity.WARNING,
+        docs="""
+        Example of rule violation::
+        
+             Keyword
+             ...  1
+             # empty line in-between multiline statement
+             ...  2
+        
+        """,
     ),
     "1014": Rule(
         rule_id="1014",
         name="variable-should-be-left-aligned",
         msg="Variable in Variable section should be left aligned",
         severity=RuleSeverity.ERROR,
         version=">=4.0",
+        docs="""
+        Example of rule violation::
+        
+            *** Variables ***
+             ${VAR}  1
+              ${VAR2}  2
+        
+        """,
     ),
     "1015": Rule(
         rule_id="1015",
         name="misaligned-continuation-row",
         msg="Each next continuation line should be aligned with the previous one",
         severity=RuleSeverity.WARNING,
+        docs="""
+        Example of rule violation::
+        
+            *** Settings ***
+            Documentation      Here we have documentation for this suite.
+            ...                Documentation is often quite long.
+            ...
+            ...               It can also contain multiple paragraphs.  # misaligned
```

Comment:
Better to make this more visible:
```suggestion
            ...            It can also contain multiple paragraphs.  # misaligned
```